addons/l10n_ch/models/account_invoice.py

```diff
@@ -215,6 +215,6 @@ def message_post(self, **kwargs):
     def _get_computed_reference(self):
         self.ensure_one()
         if self.company_id.invoice_reference_type == 'ch_isr':
-            return self.l10n_ch_isr_number
+            return self.l10n_ch_isr_number_spaced
```

Comment:
:warning: 

It would be great to show a human readable reference in there. But we shouldn't do it. :no_good_man: 
The reconciliation process splits :hocho:  the reference by spaces.


1. `_get_computed_reference` sets `invoice.reference`
2. Which is then copied to the move: `invoice.move_id.ref = invoice.reference`
https://github.com/odoo/odoo/blob/12.0/addons/account/models/account_invoice.py#L1386
3. Later the reconcile process does `invoice.move_id.ref.split(' ')` in SQL
https://github.com/odoo/odoo/blob/12.0/addons/account/models/account_reconcile_model.py#L462

Result:

An ISR such as: `"56 78901 23456 78901 23456 78901"`
Will be considered by the reconciliation process as:
`["56", "78901", "23456", "78901", "23456", "78901"]`

Thus the matching would be quite fancy. :bomb: